python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -470,6 +499,7 @@ def log_materialization(context):
         else:
             check.failed("Unexpected event {event}".format(event=event))
 
+    @public
     def add_output_metadata(
         self,
         metadata: Mapping[str, Any],
```

Comment:
I don't think we want to make this public on `OpExecutionContext`. In fact, I think we probably want to get rid of it, since you can use `Output` objects to log metadata.

Reply:
I think you are right